src/orange/Types.hh

```diff
@@ -100,6 +109,24 @@ enum class SurfaceType : unsigned char
     size_ //!< Sentinel value for number of surface types
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * Enumeration for type-deleted universe storage.
+ *
+ * See \c orange/univ/UniverseTypeTraits.hh for how these map to data and
+ * classes.
+ */
+enum class UniverseType : unsigned char
+{
+    simple,
+#if 0
+    rect_array,
+    hex_array,
+    dode_array,
+    ...
+#endif
```

Comment:
Should this be cut out or do you want to leave it in?

Reply:
I'd like to leave it in as a placeholder for @elliottbiondo 's work on arrays etc., and to indicate that we'll be adding more universe types here. (Similarly I've disabled the surfaces that aren't yet implemented.)